src/Controllers/GagePageController.ts

```diff
@@ -685,12 +695,88 @@ module StreamStats.Controllers {
             }
             if (this.dailyFlow) {
                 this.dailyFlow.forEach(dailyObj => {
-                    if (dailyObj.qualifiers[0] === 'A') {
+                    if (parseInt(dailyObj.value) !== -999999) {
                     this.formattedDailyFlow.push({x: new Date(dailyObj.dateTime), y: parseInt(dailyObj.value)})
+                    this.dailyValuesOnly.push(parseInt(dailyObj.value));
                     }
                 });
             }
-            if (this.floodFreq) { //set up AEP plotLines
+            if (this.dailyFlow) {
```

Comment:
This if statement and the if statement at 696 are the same, so they could be combined into one. If we are keeping them apart for organization purposes, maybe we could add a comment by each if statement what (or what plot) it is being used for.